tensorflow/java/src/test/java/org/tensorflow/TensorTest.java

```diff
@@ -401,71 +479,74 @@ public void failCreateOnMismatchedDimensions() {
   }
 
   @Test
-  public void failCopyToOnIncompatibleDestination() {
+  public void whenCopyToWithDimensionMismatchThenThrowExpectedException() {
```

Comment:
It is verbose more figurative than literal. I did not know what language would be used. If,if else